src/OpenLoco/src/Objects/RoadObject.h

```diff
@@ -16,42 +17,46 @@ namespace OpenLoco
         struct RenderTarget;
     }
 
-    namespace Flags12
+    enum class RoadObjectFlags : uint16_t // Made this a uint16_t to maintain the size of the RoadObject struct
     {
-        constexpr uint8_t unk_01 = 1 << 1;
-        constexpr uint8_t unk_02 = 1 << 2;
-        constexpr uint8_t unk_03 = 1 << 3; // Likely isTram
-        constexpr uint8_t unk_04 = 1 << 4;
-        constexpr uint8_t unk_05 = 1 << 5;
-        constexpr uint8_t isRoad = 1 << 6; // If not set this is tram track
-    }
+        none = 0U,
+        unk_01 = 1U << 1,
+        unk_02 = 1U << 2,
+        unk_03 = 1U << 3, // Likely isTram
+        unk_04 = 1U << 4,
+        unk_05 = 1U << 5,
+        isRoad = 1U << 6, // If not set this is tram track
+    };
+    OPENLOCO_ENABLE_ENUM_OPERATORS(RoadObjectFlags);
 
-    namespace RoadPieceFlags
+    enum class RoadObjectPieceFlags : uint16_t
     {
-        constexpr uint16_t oneWay = 1 << 0;
-        constexpr uint16_t track = 1 << 1;
-        constexpr uint16_t slope = 1 << 2;
-        constexpr uint16_t steepSlope = 1 << 3;
-        constexpr uint16_t intersection = 1 << 2;
-        constexpr uint16_t oneSided = 1 << 5;
-        constexpr uint16_t overtake = 1 << 6;
-        constexpr uint16_t streetLights = 1 << 8;
-    }
+        none = 0U,
+        oneWay = 1U << 0,
+        track = 1U << 1,
+        slope = 1U << 2,
+        steepSlope = 1U << 3,
+        intersection = 1U << 2, // This is never referenced anywhere and in the same spot as slope
```

Comment:
I don't think such comments are actually useful for readers, it may also be used in a function we haven't implemented yet I'm not sure on that, either way remove the comment please.

Reply:
To add to that sometimes the names are from LocoTool that is why we have some named but unused.